test/index.ts

```diff
@@ -120,8 +130,9 @@ const v1Override = {};
 // tslint:disable-next-line no-any
 let v1ClientOverrides: any = {};
 
-function defineOverridableClient(clientName) {
+function defineOverridableClient(clientName: string) {
   function DefaultClient() {}
+  /** @type {never[]} */
```

Comment:
What is that!?

Reply:
fix suggestion by VS code, "infer type from usage"

Reply:
In this case we generally prefer to use `any[]`.  Could you remove this and do that instead?

Reply:
done